src/5e-SRD-Features.json

```diff
@@ -33,6 +34,28 @@
     "desc": [
       "While you are not wearing any armor, your Armor Class equals 10 + your Dexterity modifier + your Constitution modifier. You can use a shield and still gain this benefit."
     ],
+    "feature_specific": {
+      "armor_class": {
+        "type": "sum",
+        "values": [
+          {
+            "type": "integer",
+            "value": 10,
+            "desc": "Base value for calculation"
```

Comment:
I feel like straight up "base" values for calculations probably don't need descriptions?

Reply:
hmm, you are right. I got too carried away with describing things